lib/formatters/load-formatter.js

```diff
@@ -4,6 +4,7 @@ import path from 'node:path';
 import JsonFormatter from './json.js';
 // The following disable should be safe. This particular rule does not need to identify
 // cycles that are broken when using dynamic imports. See https://github.com/import-js/eslint-plugin-import/issues/2265
+import KakouneFormatter from './kakoune.js';
```

Comment:
The comment is above the wrong line now.

```suggestion
import KakouneFormatter from './kakoune.js';
// The following disable should be safe. This particular rule does not need to identify
// cycles that are broken when using dynamic imports. See https://github.com/import-js/eslint-plugin-import/issues/2265
```

Reply:
Done